### DISCUSSION TITLE
Microsoft & others telll me to login here.  How is it done?  Thank You.

### DISCUSSION THREAD

**Discussion by doegie2gmailcom** [community/community#78547] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Microsoft & others telll me to login here.  How is it done?  Thank You.

**Comment by ghostinhershell** [comment]
Hi @doegie2gmailcom, we're happy you're here! You are more likely to get a useful response if you can give more information about the issue you're running into here. Giving a few more details might help someone give you a nudge in the right direction.
